src/NHSD.BuyingCatalogue.Database.PublishAll/PublishAll.dtsx

```diff
@@ -0,0 +1,4523 @@
+﻿<?xml version="1.0"?>
+<DTS:Executable xmlns:DTS="www.microsoft.com/SqlServer/Dts"
+  DTS:refId="Package"
+  DTS:CreationDate="12/12/2019 10:17:38 AM"
+  DTS:CreationName="Microsoft.Package"
+  DTS:CreatorComputerName="LPT3353"
+  DTS:CreatorName="BJSS\andy.berry"
```

Comment:
Very minor & not really sure what to do but wondering whether or not we should keep this information in i.e. BJSS\andy.berry & LPT3353?

Reply:
Removed.